deploy-manage/security/secure-settings.md

```diff
@@ -16,232 +15,177 @@ mapped_pages:
 
-    Narrow the list by name, ID, or choose from several other filters. To further define the list, use a combination of filters.
+{{es}} provides the [`elasticsearch-keystore`](elasticsearch://reference/elasticsearch/command-line-tools/elasticsearch-keystore.md) command-line tool to help with this task. It allows you to list, add, remove, and update secure settings, as well as protect the keystore with a password if wanted.
 
-3. From your deployment menu, select **Security**.
-4. From the **Existing keystores** list, use the delete icon next to the **Setting Name** that you want to delete.
-5. On the **Confirm to delete** window, select **Confirm**.
-6. All modifications to the non-reloadable keystore take effect only after restarting {{es}}. Reloadable keystore changes take effect after issuing a [reload_secure_settings](https://www.elastic.co/docs/api/doc/elasticsearch/operation/operation-nodes-reload-secure-settings) API request.
+:::{important}
+Secure settings must be specified on every node, and must have the same values across the cluster to ensure consistent behavior.
 
-:::{dropdown} Using the API
+Changes to the keystore take effect only after restarting {{es}}, except for [reloadable settings](#reloadable-secure-settings) that can be refreshed using the API.
+:::
 
-**Steps**
+**Create the keystore**
 
-Create the keystore:
+To create the `elasticsearch.keystore`, use the `create` command:
 
 ```sh
-curl -k -X PATCH -H "Authorization: ApiKey $ECE_API_KEY" https://$COORDINATOR_HOST:12443/api/v1/deployments/$DEPLOYMENT_ID/elasticsearch/$REF_ID/keystore \
-{
-  "secrets": {
-    "s3.client.CLIENT_NAME.access_key": {
-      "as_file": false
-      "value": "ACCESS_KEY_VALUE"
-    }
-    "s3.client.CLIENT_NAME.secret_key": {
-      "value": "SECRET_KEY_VALUE"
-    }
-  }
-}
+bin/elasticsearch-keystore create -p
 ```
 
-`ELASTICSEARCH_CLUSTER_ID`
-:   The {{es}} cluster ID as shown in the Cloud UI or obtained through the API
+You are prompted to enter the keystore password. A password-protected `elasticsearch.keystore` file is created alongside the `elasticsearch.yml` file.
 
-List the keys defined in the keystore:
+**List settings in the keystore**
 
-```sh
-{
-  "secrets": {
-    "s3.client.CLIENT_NAME.access_key": {
-      "as_file": false
-    },
-    "s3.client.CLIENT_NAME.secret_key": {
-      "as_file": false
-    }
-  }
-}
-```
-
-Create the credentials for an S3 or Minio repository:
+To list the settings in the keystore, use the `list` command.
 
 ```sh
-curl -k -X PUT -H "Authorization: ApiKey $ECE_API_KEY" https://$COODINATOR_HOST:12443/api/v1/clusters/elasticsearch/$ELASTICSEARCH_CLUSTER_ID/_snapshot/s3-repo
-{
-  "type": "s3",
-  "settings": {
-    "bucket": "s3_REPOSITORY_NAME",
-    "client": "CLIENT_NAME",
-    "base_path": "PATH_NAME"
-  }
-}
+bin/elasticsearch-keystore list
 ```
+If the {{es}} keystore is password protected, you are prompted to enter the password.
```

Comment:
does this mean the password can be blank? maybe we specify that above?

Reply:
Now I got your point. I've fixed it. Good catch!